### ISSUE TITLE
Adapt layout file-name pattern

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/android#4278]
According to several guidelines I think we should adapt a certain naming pattern for our files (first step) and at a later point in time also for its elements (second step), see:
* https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614
* https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md

For discussion pinging @tobiasKaminsky @ezaquarii

**Comment by tobiasKaminsky** [comment]
I am totally ok with 1.2 File naming of https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md
(the first link seems to be an excerpt of the second (or vice vera)

If we do this, this should be done one PR per type, e.g
- change drawables
- change layout file
- …

**Comment by AndyScherzinger** [comment]
> If we do this, this should be done one PR per type, e.g

👍 Absolutely

**Comment by codedsun** [comment]
Can I take this issue?

**Comment by AndyScherzinger** [comment]
Hi @codedsun, sure thing 👍 

As discussed in this issue it would be best to have a PR per type 😃

**Comment by rajput2122** [comment]
is this issue closed?

**Comment by AndyScherzinger** [comment]
@rajput2122 issue is still open.

**Comment by rajput2122** [comment]
Can i work on it?

**Comment by AndyScherzinger** [comment]
Absolutely, that be highly appreciated. Beware that files need to be renamed via git so the original authors and git history doesn't get lost and each resource type should get its very own PR to limit PR size and review efforts :)
